src/usecases.js

```diff
@@ -3,7 +3,13 @@ const list = require('./list/list')
 const usecases = new Map()
 
 const builder = (usecase, id) => {
-    const defaultItem = item(id)
+    const usecaseId = id || usecase.description
```

Comment:
How does this function handle use case names with spaces? The correct thing would be to generate names without spaces and pascal case.